### PR TITLE
added laravel/pint & configured as linter and formatter

### PR DESCRIPTION
**Pull request by MagnaibayarMN**
- added laravel/pint package into composer.json
- added composer lint & composer format
- added .github/workflow from https://github.com/utopia-php/ab/tree/feat-code-formatter/.github

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,8 +18,13 @@
...
         "psr-4": {"Utopia\\System\\": "src/System"}
     },
+    "scripts": {
+      "lint": "./vendor/bin/pint --test",
+      "format": "./vendor/bin/pint"
+    },
     "require": {
-        "php": ">=7.4"
+        "php": ">=7.4",
+        "laravel/pint": "^1.2"
```
Comment (Meldiron) [inline composer.json:27]:
```suggestion
        "laravel/pint": "dev-main"
```

Let's please use this version to stay consistent with all other repositories.

**Review by Meldiron — CHANGES_REQUESTED** [review]
Thanks for the PR 😇 I left a comment, let' please address it.

Also, please run `composer format`, which will apply linting to all existing PHP files. That will make a bunch of changes, please commit those too - this is necessary for tests to pass.

File: composer.json (hunk 1 above)
Comment (MagnaibayarMN) [reply composer.json]: Thank you, I fixed them. Could you please check it out.

**Review by Meldiron — APPROVED** [review]
Looks great 🎉

File: composer.json [hunk 2]
```diff
@@ -18,8 +18,13 @@
...
         "psr-4": {"Utopia\\System\\": "src/System"}
     },
+    "scripts": {
+      "lint": "./vendor/bin/pint --test",
+      "format": "./vendor/bin/pint"
+    },
     "require": {
-        "php": ">=7.4"
+        "php": ">=7.4",
+        "laravel/pint": "dev-main"
```
Comment (christyjacob4) [inline composer.json:27]:
```suggestion
        "laravel/pint": "1.2.*"
```

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**